docs/ovndb_ha.md

```diff
@@ -0,0 +1,56 @@
+# OVN Northbound/Southbound DB HA using Corosync/Pacemaker
+
+The following document captures how to start OVN DBs in Active/Standby
+HA mode using Corosync and Pacemaker. This HA mode is supported by OVS
+community and is widely used. For more information on how the HA itself
+is done please read the information here:
+
+http://docs.openvswitch.org/en/latest/topics/integration/#ha-for-ovn-db-servers-using-pacemaker
+
+The following diagram captures the setup for this type of HA. The ovnkube-db
+deployment has replica set to 3 and the ovnkube-db pods will end up on those
+nodes that have label set to openvswitch.org/ovnkube-db=true.
+
+![alt-txt](ovndb_ha_active_stdby.svg)
+
+The OVN NB/SB DB will be available behind a Virtual IP which should be from the
+K8s Node subnet and will be provided by the user. Both the Northbound/Southbound
+DBs run in a single container in the ovnkube-db pod. This is done so that it
+simplifies setting up corosync/pacemaker as we just need one cluster to run both
```

Comment:
"so that it simplifies" should be "to simplify"
"as" should be "since"
Also, "cluster" to a kubernetes user means a kubernetes cluster. This should have an adjective in front of cluster.